whois-api/src/main/java/net/ripe/db/whois/api/exceptions/UpgradeException.java

```diff
@@ -0,0 +1,12 @@
+package net.ripe.db.whois.api.exceptions;
+
+import javax.ws.rs.WebApplicationException;
+
+public class UpgradeException extends WebApplicationException {
+
+    private static final long serialVersionUID = -6362082162191216040L;
+
```

Comment:
Are you sure it will return 429? There is NrtmExceptionMapper and it returns InternalServer error. Can you add a test case just to be sure?

Reply:
Also I don't think you need a custom exception for this. Throwing SC_UPGRADE_REQUIRED status should be enough

Reply:
there are 2 test case in which we are returning 426 I added this exception in the mapper as well

Reply:
I added the exception because I think that it is more clear...we are using notfoundexception, badrequestexception from WebApplicationException...so create a new one which is not already implemented by the library is the correct way (to follow the same behaviour)